appveyor/build_task.cmd

```diff
@@ -1,7 +1,7 @@
 @echo off
 setlocal enableextensions enabledelayedexpansion
-	wget -N --progress=bar:force:noscroll http://windows.php.net/downloads/php-sdk/deps-%PHP_REL%-vc14-%PHP_SDK_ARCH%.7z -P %CACHE_ROOT%
-	7z x -y %CACHE_ROOT%\deps-%PHP_REL%-vc14-%PHP_SDK_ARCH%.7z -oC:\projects\php-src
+	wget -N --progress=bar:force:noscroll http://windows.php.net/downloads/php-sdk/deps-%PHP_REL%-%PHP_BUILD_CRT%-%PHP_SDK_ARCH%.7z -P %CACHE_ROOT%
```

Comment:
This command not work actually for the PHP 7.2 branch because the file `deps-7.2-vc15-x86.7z` is not found.